.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
   "typescript.tsdk": "node_modules/typescript/lib",
   "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
why did you made a change here?

Reply:
actually i did not its happend automatically i dont know the reason